File: benchmark/results/ts-script-sonnet/run-004/b-baseline/output/src/domain/billing/services/TaxCalculator.ts
import { Invoice } from '../aggregates/Invoice';

/** Calculator for computing tax amounts on invoices. */
export class TaxCalculator {
  create(input: Partial<Invoice>): Invoice {
    void input;
    throw new Error('not implemented');
  }

  findById(id: string): Invoice | null {
    void id;
    throw new Error('not implemented');
  }

  list(limit: number): Invoice[] {
    void limit;
    throw new Error('not implemented');
  }

  delete(id: string): void {
    void id;
    throw new Error('not implemented');
  }
}

File: benchmark/results/python-multilang/run-004/b-baseline/output/metaengine_demo/billing/services/tax_calculator.py
"""TaxCalculator for the billing domain."""
from typing import Optional

from metaengine_demo.billing.aggregates.invoice import Invoice


class TaxCalculator:
    """Calculator for invoice taxes."""

    def create(self, input: Optional[Invoice]) -> Invoice:
        """Create tax data for a new Invoice."""
        raise NotImplementedError("not implemented")

    def findById(self, id: str) -> Optional[Invoice]:
        """Find tax-bearing Invoice by id."""
        raise NotImplementedError("not implemented")

    def list(self, limit: float) -> list[Invoice]:
        """List tax-bearing Invoices up to limit."""
        raise NotImplementedError("not implemented")

    def delete(self, id: str) -> None:
        """Delete tax data for an Invoice by id."""
        raise NotImplementedError("not implemented")
